acceptance-tests/src/main/kotlin/org/ostelco/at/jersey/Tests.kt

```diff
@@ -283,6 +283,59 @@ class RegionsTest {
             StripePayment.deleteCustomer(customerId = customerId)
         }
     }
+    @Test
+    fun `jersey test - POST support simprofile - Sim profile from houston`() {
+
+        val email = "regions-${randomInt()}@test.com"
+        var customerId = ""
+        try {
+            customerId = createCustomer(name = "Test Single Region User", email = email).id
+            enableRegion(email = email)
+
+            val dataMap = MultivaluedHashMap<String, String>()
+            dataMap["regionCode"] = listOf("no")
+            dataMap["profileType"] = listOf("iPhone")
+
+            var regionDetailsList: Collection<RegionDetails> = get {
+                path = "/regions"
+                this.email = email
+            }
+
+            assertEquals(2, regionDetailsList.size, "Customer should have 2 regions")
+            var receivedRegion = regionDetailsList.find {
+                it.status == APPROVED
+            }!!
```

Comment:
I don't know if kotlin will  propagate assertNotNull done on a variable as that variable being not null.  If that is true, then it could be used instead of "!!", since it will bring a little more clarity to the code, and also more clarity to an error message, should it ever happen.